.github/workflows/changelog.yml

```diff
@@ -1,6 +1,9 @@
 name: Changelog
 
 on:
+  workflow_dispatch:
+  release:
+    types: [created]
```

Comment:
I normally create releases on my GH projects, so kind of like it when the changelog auto updates for me.